pom.xml

```diff
@@ -76,6 +76,16 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>2.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.freemarker</groupId>
```

Comment:
Why do you have added this dependency? If not strictly required for the PR I suggest to remove it (and in case create another PR ...).